tokens.html

```diff
@@ -2,27 +2,31 @@
 title: Tokens
 layout: default
 permalink: /tokens/
+subnav:
+  font-size: font-size
```

Comment:
Can we use capitalized versions for the visible sub nav links?
```suggestion
  font-size: Font Size
```